proxy-attestation-server/Cargo.toml

```diff
@@ -49,7 +49,7 @@ http = "=0.2.4"
 lazy_static = "1.3"
 log = "=0.4.13"
 nb-connect = "=1.0.3"
-nitro-enclave-attestation-document = { git = "https://github.com/veracruz-project/nitro-enclave-attestation-document.git", branch = "main", optional = true }
+nitro-enclave-attestation-document = { git = "https://github.com/veracruz-project/nitro-enclave-attestation-document.git", rev = "4d703bcde209a8d625776f6c346c36a9ad084511", optional = true }
```

Comment:
This will probably break at some point

Reply:
This essentially reverts https://github.com/veracruz-project/nitro-enclave-attestation-document/commit/0b97cfca8d0087592efb85e142ed2ee9607ad60b to allow making a different version pinning in Veracruz itself. `nitro-enclave-attestation-document` pins `serde_with` to version 1.9.4 which requires `serde` version 1.0.122 or later. As long as we are using our vendored version of `serde` (circa 1.0.115) this change will be needed for workspaces. Once we move to upstream `serde` we can go back to `main` branch (and probably remove the pin in `nitro-enclave-attestation-document`).